sdk/appplatform/Azure.ResourceManager.AppPlatform/src/autorest.md

```diff
@@ -282,11 +282,11 @@ rename-mapping:
   ResourceSku.locations: -|azure-location
   ResourceSkuRestrictionInfo.locations: -|azure-location
 
+parameter-rename-mapping:
+  ConfigServers_Validate:
+    configServerSettings: settings
+
 directive:
-  - from: swagger-document
```

Comment:
How could this file be changed?

Reply:
I changed this. Some directives here are redundant.